Libraries/src/Amazon.Lambda.Annotations.SourceGenerator/ProjectFileHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿using Amazon.Lambda.Annotations.SourceGenerator.FileIO;
+using System;
+using System.Xml;
+
+namespace Amazon.Lambda.Annotations.SourceGenerator
+{
+    /// <summary>
+    /// Utilities for working with project (.csproj) files, such as allowing projects
+    /// to opt out of the CloudFormation description modification.
+    /// </summary>
+    public class ProjectFileHandler
+    {
+        private const string OptOutNodeXpath = "//PropertyGroup/AWSSuppressCloudFormationDescriptionModification";
+
+        /// <summary>
+        /// Determines if the project has opted out of modifying the CloudFormation template descriptio
+        /// </summary>
+        /// <param name="projectFilePath">Path to a .csproj file</param>
+        /// <param name="fileManager">FileManager instance used to read the csproj contents</param>
+        /// <returns>True if opted out of modifying the description, false otherwise</returns>
+        public static bool IsCloudFormationDescriptionModificationSuppressed(string projectFilePath, IFileManager fileManager)
```

Comment:
nit: same comment about renaming this method

Reply:
Fixed in amended commit.